pkg/sfu/downtrack.go

```diff
@@ -19,6 +20,19 @@ import (
 	"github.com/livekit/livekit-server/pkg/sfu/buffer"
 )
 
+// TrackSender defines a  interface send media to remote peer
+type TrackSender interface {
+	UptrackLayersChange(availableLayers []uint16, layerAdded bool) (int32, error)
+	WriteRTP(p *buffer.ExtPacket, layer int32) error
```

Comment:
Use layer 0 for simple track, because most code of process simple & simulcast packet is same. so I preferred a single method for the two tracks.